scripts/SANS/sans/gui_logic/models/table_model.py

```diff
@@ -309,6 +309,10 @@ def to_list(self):
                 self.options_column_model.get_options_string()]
 
     def _convert_sample_shape_to_string(self, shape):
+        if isinstance(shape, str):
+            # TODO temporary fix to shape already being a str
+            # find out where this is being converted as we are trying to convert twice
+            return shape
```

Comment:
Is there an issue for this? Would it be worth opening one?

Reply:
#24261 now open